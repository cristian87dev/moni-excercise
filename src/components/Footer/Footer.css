.footer {
  width: 100%;
  padding: 1rem;
  background-color: var(--bg-color);
}

.legend {
  margin: 0;
  text-align: right;
  color: var(--font-color);
  font-size: 0.8rem;
}
