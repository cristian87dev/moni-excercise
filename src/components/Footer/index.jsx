import React from 'react';

import './Footer.css';

const Footer = () => (
  <footer className="footer">
    <p className="legend">Cristian Perrone for Moni&trade;</p>
  </footer>
);

export default Footer;
