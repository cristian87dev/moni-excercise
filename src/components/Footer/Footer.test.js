import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './index';

it('Footer renders without crashing', () => {
  const component = renderer.create(
    <Footer />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
