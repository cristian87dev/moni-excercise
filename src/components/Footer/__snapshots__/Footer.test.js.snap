// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders without crashing 1`] = `
<footer
  className="footer"
>
  <p
    className="legend"
  >
    Cristian Perrone for Moniâ„¢
  </p>
</footer>
`;
